examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -89,7 +88,7 @@
  *   --output=gs://YOUR_OUTPUT_PREFIX
  * }</pre>
  *
- * <p>The input file defaults to {@code gs://apache-beam-samples/shakespeare/kinglear.txt}
+ * <p>The input file defaults to {@code gs://apache-beam-samples/apache/LICENSE}
```

Comment:
I would also love to revert the default input file to shakespeare. (Set the LICENSE file as input in WordCountIT.)